src/oop/programs_with_multiple_classes/medium/q3/QuestionText.md

```diff
@@ -0,0 +1,39 @@
+# oop-programs_with_multiple_classes-medium-q1
```

Comment:
Also, I think we should add a marker in this class so we know what kind of question it is (Parsons, Java, etc.).